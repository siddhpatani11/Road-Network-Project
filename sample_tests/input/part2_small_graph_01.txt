13 83
0 3 8 R
0 7 27 R
0 9 28 G
1 0 37 R
1 2 42 G
1 3 49 R
1 4 29 B
1 5 41 R
1 5 47 G
1 10 16 R
1 11 22 R
1 11 23 G
2 3 18 G
2 7 41 B
2 9 11 R
2 12 4 G
2 12 37 R
3 1 34 B
3 4 20 B
3 8 50 R
3 9 40 B
3 9 2 R
3 11 24 B
3 12 50 G
4 5 15 G
4 6 39 B
4 7 12 G
4 8 17 B
4 8 48 B
5 0 1 B
5 0 37 B
5 1 5 R
5 2 48 B
5 6 26 R
5 7 9 G
5 9 10 B
5 10 14 R
6 0 49 R
6 4 51 B
6 7 39 R
6 11 47 G
7 4 21 R
7 6 22 G
7 10 39 G
7 10 25 R
7 12 7 R
7 12 12 R
8 4 47 G
9 2 31 R
9 2 15 G
9 3 49 R
9 4 5 R
9 5 8 B
9 5 20 R
9 6 27 B
9 6 49 G
9 8 31 G
9 12 6 R
10 2 33 G
10 3 2 R
10 3 30 R
10 6 27 B
10 6 2 B
10 7 38 B
10 8 8 B
10 11 34 R
11 2 7 R
11 2 41 B
11 3 15 G
11 4 45 R
11 8 6 G
11 9 44 R
11 10 18 B
12 1 22 R
12 2 30 G
12 3 10 R
12 3 37 R
12 7 39 R
12 7 38 R
12 8 21 R
12 8 24 G
12 9 5 B
12 11 32 R
