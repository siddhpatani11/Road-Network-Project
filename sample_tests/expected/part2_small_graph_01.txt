9 R 21 B 13 G 15 R 29 R 5 B 8 B 27 R 17 R 30 - 0 B 22 B 36 R 6 
